tools/mmp/driver.cs

```diff
@@ -800,6 +808,14 @@ static void Pack (IList<string> unprocessed)
 			if (App.LinkMode != LinkMode.All && App.RuntimeOptions != null)
 				App.RuntimeOptions.Write (App.AppDirectory);
 
+			if (aot.IsAOT) {
+				if (!IsUnified || !Is64Bit)
+					throw new MonoMacException (98, true, "AOT compilation is only available on Unified 64-bit", ret);
```

Comment:
I don't think it will - but we ship two (libmono) runtimes (both 32 and 64) so building/shipping our own mono32 is not much additional work. FWIW we're shipping a lot more for XI :)